MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -35,7 +35,7 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     func handleBridgeMessage(responseJson: MiniAppJavaScriptMessageInfo?) {
         guard let actionCommand = responseJson?.action, !actionCommand.isEmpty,
             let callbackId = responseJson?.id, !callbackId.isEmpty else {
-            executeJavaScriptCallback(responseStatus: .onError, messageId: "", response: MiniAppJavaScriptError.unexpectedMessageFormat.rawValue)
+                executeJavaScriptCallback(responseStatus: .onError, messageId: "", response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
```

Comment:
This means that the error is still going to be received as a string by the JS SDK, right? 

I think that we should be directly passing an [error object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error). So maybe we need to update the JS Bridge code (i.e. here: https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-bridge/src/common-bridge.ts#L191) so that it parses the string and creates an Error object? Something like this (in typescript):
```
class MiniAppError extends Error {
    constructor(errorResponse: string) {
        super();
        const error = JSON.parse(errorResponse);
        this.name = error.name;
        this.message = error.message;
    }    
}
```

Reply:
Got it 👍 , will create PR in JS SDK

Reply:
I guess this change is no longer needed.